lib/openc_bot/helpers/incremental_search.rb

```diff
@@ -102,6 +142,10 @@ def increment_number(uid, increment_amount = 1)
       def max_failed_count
         const_defined?("MAX_FAILED_COUNT") ? const_get("MAX_FAILED_COUNT") : 10
       end
+
+      def max_gap_count
+        const_defined?("MAX_GAP_COUNT") ? const_get("MAX_GAP_COUNT") : 10
```

Comment:
Is 10 too small? The gap fetching will ignore the exist data so I think we will need a big number